src/models.py

```diff
@@ -0,0 +1,36 @@
+from flask_sqlalchemy import SQLAlchemy
+from flask_migrate import Migrate
+
+import auth_util
+
+db = SQLAlchemy()
+migrate = Migrate()
+
+class User(db.Model):
+    user_id = db.Column(db.Integer, primary_key=True)
```

Comment:
Think we should just go with `id` for primary_key ids. Otherwise, it just becomes redundant doing `Model.model_id`.